modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -50,7 +50,11 @@ sda.enkf.multisite <- function(settings,
   }else{
     restart_flag = FALSE
   }
-  future::plan(multiprocess)
+  if (future::supportsMulticore()) {
+    future::plan(future::multicore)
+  } else {
+    future::plan(future::multisession)
+  }
```

Comment:
Apologies if this is documented somewhere already: Does this function provide a way for the user to control how many processes it runs in parallel? The ability to use cores when they're available is great, but having something use lots of cores when you didn't expect it is a lot less great, and CRAN is firm that package code should only run multicore when the user specifically tells it to.

Reply:
Fixed. I added a new argument called `run_parallel` in the SDA function.